### PR TITLE
Fix enclave setup not overwriting the previous config

### PR DESCRIPTION
**Pull request by Piccirello**

### CODE REVIEW

File: pkg/cmd/enclave_setup.go [hunk 1]
```diff
@@ -116,17 +116,23 @@ var setupCmd = &cobra.Command{
...
+			var options []string
+			var defaultOption string
 			for _, val := range configs {
-				configOptions = append(configOptions, val.Name)
+				option := val.Name
+				options = append(options, option)
+				// reselect previously-configured value
+				if val.Name == scopedConfig.EnclaveConfig.Value {
+					defaultOption = val.Name
+				}
```
Comment (bvallelunga) [inline pkg/cmd/enclave_setup.go]: Why should we care about the previous value?

Comment (Piccirello) [reply pkg/cmd/enclave_setup.go]: So that we can re-select the previous value when re-running setup. I believe this was a feature you requested.

Comment (bvallelunga) [reply pkg/cmd/enclave_setup.go]: Gotcha, can we only re-select the config if the project is the same.

Comment (Piccirello) [reply pkg/cmd/enclave_setup.go]: I like that

Comment (Piccirello) [reply pkg/cmd/enclave_setup.go]: Addressed

**Review by bvallelunga — APPROVED** [review]

**Pull request merged**